docs/_future/app_manifests.md

```diff
@@ -0,0 +1,106 @@
+---
+title: App manifest
+order: 3
+slug: app-manifest
+lang: en
+layout: tutorial
+permalink: /future/app-manifest
+---
+# App manifests <span class="label-beta">BETA</span>
+
+<div class="section-content">
+An app's manifest is where you can configure its name and scopes, declare the functions your app will use, and more. 
+</div>
+
+---
+
+### Configuring an app {#configuring-an-app}
+
+Locate the file named `manifest.js` within your application. This will likely be in your project's root directory or a `manifest` folder. 
+
+Inside the manifest file, you will find an `module.exports = Manifest({})` block that defines the app's configuration:
+
+```javascript
+// manifest/manifest.js
+
+const { Manifest } = require('@slack/bolt');
+const { TimeOffWorkflow } = require('./workflow/approval');
+
+module.exports = Manifest({
+  runOnSlack: false,
+    // This is the internal name for your app. 
+  // It can contain spaces (e.g., "My App")
+  name: 'take-your-time',
+  displayName: 'Take Your Time',
+  // A description of your app that will help users decide whether to use it.
+  description: 'Request and take time off.',
+  longDescription: 'Take your time off by using this application to request and take time off from your manager. Launch the workflow, put in your manager, requested PTO start and end date, and receive updates on your PTO request!',
+  botScopes: ['chat:write'],
+  tokenManagementEnabled: true,
+  socketModeEnabled: true,
+  // A list of all workflows your app will use.
+  workflows: [TimeOffWorkflow],
+  features: {
+    appHome: {
+      homeTabEnabled: true,
+      messagesTabEnabled: true,
+      messagesTabReadOnlyEnabled: true,
+    },
+  },
+  settings: {
+    interactivity: {
+      is_enabled: true,
+    },
+  },
+  tokenRotationEnabled: false,
+});
+```
+---
+
+### Manifest properties {#manifest-properties}
+
+The properties in the Manifest are:
+
+|Property|Type|Description|
+|---|---|---|
+| `name` | String | This is the internal name for your app. It can contain spaces (e.g., "My App") |
+| `description` |String| A short sentence describing your application. A description of your app that will help users decide whether to use it |
+| `displayName` | String | (Optional) Allows a custom name for the app to be displayed that's different from the `name` |
+| `longDescription` | String | (Optional) A more detailed description of your application |
+| `icon` | String | A relative path to an image asset to use for the app's icon. Your app's profile picture that will appear in the Slack client |
```

Comment:
This doesn't work currently - appears to only work on `slack deploy` for Deno Apps. Can we remove this field, or better mark that it's unsupported in Bolt currently